### ISSUE TITLE
how to define positive samples?

### ISSUE THREAD

**Issue by 2anchao** [xingyizhou/CenterNet2#66]
## 📚 Documentation

* Links to the relevant documentation/comment:
Hi
code:

       pos_inds, labels, reg_targets, flattened_hms = \
            self._get_ground_truth(
                grids, shapes_per_level, gt_instances)`

        logits_pred, reg_pred, agn_hm_pred = self._flatten_outputs(
            clss_per_level, reg_pred_per_level, agn_hm_pred_per_level)`

        if self.more_pos:
            # add more pixels as positive if \
            #   1. they are within the center3x3 region of an object
            #   2. their regression losses are small (<self.more_pos_thresh)
            pos_inds, labels = self._add_more_pos(
                reg_pred, gt_instances, shapes_per_level)`

questions:
why has been use self._get_ground_truth get pos_inds(I call as pos_inds1), and when self.more_pos == True, use self._add_more_pos to get pos_inds(I call as pos_inds2), so the pos_inds2 cover the pos_inds1, I think pos_inds = pos_inds1 + pos_inds2 is right?

**Comment by 2anchao** [comment]
I carelessness,


> ## books Documentation
> 
>     * Links to the relevant documentation/comment:
>       Hi
>       code:
>       ```
>        pos_inds, labels, reg_targets, flattened_hms = \
>             self._get_ground_truth(
>                 grids, shapes_per_level, gt_instances)`
>       
>         logits_pred, reg_pred, agn_hm_pred = self._flatten_outputs(
>             clss_per_level, reg_pred_per_level, agn_hm_pred_per_level)`
>       
>         if self.more_pos:
>             # add more pixels as positive if \
>             #   1. they are within the center3x3 region of an object
>             #   2. their regression losses are small (<self.more_pos_thresh)
>             pos_inds, labels = self._add_more_pos(
>                 reg_pred, gt_instances, shapes_per_level)`
>       ```
> 
> 
> questions:
> why has been use self._get_ground_truth get pos_inds(I call as pos_inds1), and when self.more_pos == True, use self._add_more_pos to get pos_inds(I call as pos_inds2), so the pos_inds2 cover the pos_inds1, I think pos_inds = pos_inds1 + pos_inds2 is right?

I carelessness, the  pos_inds1 is None when self.more_pos == True
